samples/cellular/modem_shell/src/at/at_cmd_mode_custom.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <sys/types.h>
+
+#include <zephyr/net/net_ip.h>
+#include <zephyr/shell/shell.h>
+#include <modem/at_cmd_parser.h>
+#include <modem/at_cmd_custom.h>
+#include <nrf_modem_at.h>
+
+#include "mosh_defines.h"
+#include "mosh_print.h"
+
+#if defined(CONFIG_MOSH_PING)
+#include "icmp_ping.h"
+#endif
+
+#define MOSH_AT_CMD_MAX_PARAM 8
+
+extern bool at_cmd_mode_dont_print;
+
+#if defined(CONFIG_MOSH_PING)
```

Comment:
I know this is in old code but would probably be cleaner to exclude the entire file in `CMakeLists.txt`

Reply:
I wanted to do it like this because this module is not ping specific. This module can contain handlers for different custom AT commands. It just happens to contain only ping at the moment. 😄